docker/entrypoint.sh

```diff
@@ -14,11 +14,16 @@ then
     exit 1
 fi
 
+if [ -z "$OCTOFARM_PORT" ]
+then
+    echo "OCTOFARM_PORT=$OCTOFARM_PORT is not defined, the default of 4000 will be assumed. You can override this at any point."
```

Comment:
Nice instruction for docker users. Personally I like it when containers log this kind instruction